cookbooks/travis_internal_base/recipes/default.rb

```diff
@@ -29,7 +29,7 @@
 include_recipe 'travis_sudo'
 
 package 'whois' do
-  action [:install, :upgrade]
+  action %i(install upgrade)
```

Comment:
It's a silly change on my part, imho. The resulting array of symbols is the same.